PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/security/custom/CustomBasicAuthenticationFilter.java

```diff
@@ -44,7 +44,7 @@ public CustomBasicAuthenticationFilter(final AuthenticationManager authenticatio
     protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response,
                                     final FilterChain chain) throws IOException, ServletException
     {
-        final String header = request.getHeader(modelPropertiesService.securityHeaderName);
+            final String header = request.getHeader(modelPropertiesService.securityHeaderName);
```

Comment:
xD

Reply:
Probably pressed tab while debugging. Fixed in 244b7fb